operations/mimir-mixin/dashboards/ruler.libsonnet

```diff
@@ -28,7 +28,7 @@ local filename = 'mimir-ruler.json';
       )
       .addPanel(
         $.panel('Read from ingesters - QPS') +
-        $.statPanel('sum(rate(cortex_ingester_client_request_duration_seconds_count{%s, operation="/cortex.Ingester/QueryStream"}[$__rate_interval]))' % $.jobMatcher($._config.job_names.ruler), format='reqps')
```

Comment:
Note to reviewers: I've preserved the ruler job names so that the dashboard works when deployed in monolithic and read-write mode as well as microservices mode.

Reply:
another idea is to pass an array to `jobMatcher`. You can reflect on the value of the arg in `jobMatcher` with `std.isArray(job)` and loop over arrays or just use a single string (as it is now)

Reply:
I've implemented Dimitar's suggestion and updated all `job_names` values to be arrays to make them easier to work with. Let me know what you think.